libs/domain/picking/customer-note-modal/customer-note-modal.def.ts

```diff
@@ -1,7 +1,7 @@
 import { componentDef } from '@spryker-oryx/utilities';
 
 export const customerNoteModalComponent = componentDef({
```

Comment:
Each components class name, def funcs, schema should start with domain name, ie: `pickingCustomerNoteModalComponent`, `PickingCustomerNoteModalComponent`

Double check, please, other components and make re-namings